go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/sirupsen/logrus v1.2.0 // indirect
 	github.com/stretchr/testify v1.2.2
 	github.com/sylabs/scs-library-client v0.3.0
-	github.com/sylabs/singularity v0.0.0-20190721223548-90e847a3fc6a
+	github.com/sylabs/singularity v0.0.0-20190725211753-a066ffcb65c2
```

Comment:
can't we bind to the specific version?

Reply:
My assumption is that `go mod` will change version to commit here because in Singularity repo there are non semver tags.

Reply:
doesn't it look like semver tag? https://github.com/sylabs/singularity/releases/tag/v3.3.0

Reply:
:(